.github/workflows/release-pr.yml

```diff
@@ -4,6 +4,15 @@ name: Update release notes
 # or API.
 on: 
    workflow_dispatch:
+      inputs:
+         time_range_hours:
+           description: 'SDK released within x hours'     
+           required: true
+           default: '24'
+         debug_logging:
```

Comment:
Can we make debug logs enabled by default and get away with this configuration? If not, can you make it a simple checkbox.

Reply:
The debug option is removed, and the CI job now prints debug logs by default.